test/mlir/onnx/onnx_constprop.mlir

```diff
@@ -1338,3 +1338,22 @@ func.func @test_range_fp() -> tensor<3xf32> {
 // CHECK:           onnx.Return [[VAR]] : tensor<3xf32>
 // CHECK:         }
 }
+
+// -----
+
+func.func @test_nonzero() -> tensor<2x?xi64> {
+  %0 = "onnx.Constant"() {value = dense<[[2, 1], [0, 2], [0, 1]]> : tensor<3x2xi8>} : () -> tensor<3x2xi8>
+  %1 = "onnx.NonZero"(%0) : (tensor<3x2xi8>) -> tensor<2x?xi64>
+  // NOTE: We cannot use return (i.e. "func.return") because the change of shape makes
+  // func::ReturnOp::verify() fail with:
+  //
+  //   error: type of return operand 0 ('tensor<2x4xi64>') doesn't match function result type ('tensor<2x?xi64>') in function @test_nonzero
+  //
```

Comment:
Looks like this note can be removed now, it is not about this test anymore.

Reply:
good point

removed